webpack/webpack.common.js

```diff
@@ -425,112 +434,31 @@ module.exports = () => {
-                                            }
-                                        }
-                                        catch (e) {
-                                            failureCount += 1;
-                                            console.warn(
-                                                '\x1b[33m%s\x1b[0m',  // yellow
-                                                `"${templatePath}" has failed to load. Retrying (${failureCount} / 5)...`
-                                            );
-                                            return await renderTemplate(failureCount = failureCount);
-                                        }
-                                    }
-                                    else {
-                                        if (!isTestEnvironment) {
-                                            loaderContext.emitError(Error(`Unable to fetch ${templatePath} from the Django server.`))
-                                        }
-                                        else {
-                                            console.warn(
-                                                '\x1b[31m%s\x1b[0m',  // red
-                                                `"${templatePath}" has failed to load! Test environment detected, falling back to un-rendered file.`
-                                            );
-                                        }
-
-                                        resp = {
-                                            text: () => (
-                                                new Promise((resolve, _reject) => {
-                                                    /*
-                                                        if run in a test environment, failures will return a empty string which will
-                                                        still allow the bundle to build.
-                                                    */
-
-                                                    resolve(isTestEnvironment ? '' : content);
-                                                })
-                                            )
-                                        };
-                                    }
-                                };
-
-                                await renderTemplate();
-
-                                const responseText = await resp.text();
-                                return responseText;
-                            }
-                        }
+                            publicPath: '',  // this ensures that Knockout can render the template on page load
+                            emit: false,
+                        },
                     },
                     {
                         test: /\.(txt|DS_Store)$/i,
-                        exclude: /node_modules/,
-                        use: Path.join(PROJECT_RELATIVE_NODE_MODULES_PATH, 'raw-loader'),
+                        type: 'asset/resource',
```

Comment:
general cleaning to exclude text files ( also those odd DS_Store mac files ) from being included in the bundle, instead of essentially just ignoring them